### ISSUE TITLE
The ListViewItem has the wrong position in the Inspect tree after changing the Group of the ListViewItem.

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#4847]
.NET Core Version: 
- 6.0.100-preview.2.21155.3

Have you experienced this same bug with .NET Framework?: 
- Yes

**Minimal repro:**

1. Create an app with ListView
2. Add "Test" ListViewGroup to the ListView
3. Add three ListViewItems to a ListView
4. Set the "Test" ListViewGroup as the Group for the second ListViewItem
5. Add an event so that when you click on a special button, the second ListViewItem is removed from the "Test" ListViewGroup
6. Run the application
7. Run Inspect
8. Press a special button (the second 'ListViewItem' is moved to the default ListViewGroup)

**Actual behavior:**
The item is displayed second in the Inspect tree, although it is third in the ListView
![4847](https://user-images.githubusercontent.com/23376742/116056578-9b85d580-a686-11eb-9e35-3de2b8b96d77.png)

**Expected behavior**
The ListViewItem should have the same position in the Inspect tree as in the ListView

**Comment by vladimir-krestov** [comment]
The same as #4851.
@Olina-Zhang, please test it via AI and Inspect with updating (like you tested in #4851).
If it works as expected, please close the issue

**Comment by Olina-Zhang** [comment]
Tested this issue, cannot repro in Accessibility Insight tool and Inspect with updating, so close it.
